api-reference/20 Data Visualization Widgets/BaseChart/1 Configuration/customizeLabel.md

```diff
@@ -46,6 +46,10 @@ The **customizeLabel** function accepts an object providing information about th
                 <td style="text-align:left">index</td>
                 <td>The index of the series point in the points array.</td>
             </tr>
+            <tr>
+                <td style="text-align:left">data</td>
+                <td>An object that contains the <a href="/Documentation/ApiReference/Data_Visualization_Widgets/dx{WidgetName}/Chart_Elements/Point/Fields/#data">series point data</a>.</td>
```

Comment:
Instead of this part of the link:

`/Documentation/ApiReference/Data_Visualization_Widgets/dx{WidgetName}`

you can use `{basewidgetpath}`. As a general rule, if you find that you need `{WidgetName}` in a link, most probably you can use `{basewidgetpath}` instead.